src/consumption.cpp

```diff
@@ -161,6 +219,13 @@ edible_rating player::can_eat( const item &food, bool interactive, bool force )
         return INEDIBLE;
     }
 
+    for( const auto &m : food.type->materials ) {
+        if( !m.obj().edible() ) {
```

Comment:
Comestibles should always have at least one of: positive nutrition, positive quench, use function.

A "comestible" that doesn't have any of those should be turned generic. After all such false comestibles are fixed, there should be a debugmsg on future false comestibles.

It would be even better if meds weren't comestibles in any way. It only causes problems.